agent/bench-scripts/pbench-user-benchmark

```diff
@@ -62,6 +65,13 @@ while true; do
 			shift;
 		fi
 		;;
+                --collect_sysinfo)
+                shift;
+                if [ -n "$1" ]; then
+                       collect_sysinfo="$1"
```

Comment:
Can you add code here to ensure the `collect_sysinfo` variable is one of the accepted values?  Perhaps we could modify `pbench-collect-sysinfo` so that it can be invoked with a `--check` option to verify that string?